launch_testing_ros/test/examples/check_msgs_launch_test.py

```diff
@@ -79,5 +80,5 @@ def start_subscriber(self):
         self.ros_spin_thread = Thread(target=lambda node: rclpy.spin(node), args=(self,))
```

Comment:
I can't write this up there because of GitHub limitations, but the `__init__` method should have an annotation on the `name` parameter.